test/e2e/e2e_helpers.go

```diff
@@ -131,144 +131,6 @@ func RevertKubeCtx(originalKubeContext string) {
-				"will not deploy cert-manager and they will also not undeploy it after running the test suite.\n",
-		)
-		Expect(installCertManager()).To(Succeed())
-		return true
-	} else {
-		fmt.Fprint(GinkgoWriter,
-			"The cert-manager namespace exists, assuming cert-manager has been deployed already.\n",
-		)
-	}
-	return false
-}
-
-func installCertManager() error {
-	repoList, err := Run(exec.Command("helm", "repo", "list"))
-	if err != nil {
-		return err
-	}
-	if !strings.Contains(repoList, "jetstack") {
-		fmt.Fprintf(GinkgoWriter, "The helm repo for cert-manager has not been found, adding it now.\n")
-		if err := RunAndIgnoreOutput(
-			exec.Command(
-				"helm",
-				"repo",
-				"add",
-				"jetstack",
-				"https://charts.jetstack.io",
-				"--force-update",
-			)); err != nil {
-			return err
-		}
-		fmt.Fprintf(GinkgoWriter, "Running helm repo update.\n")
-		if err = RunAndIgnoreOutput(exec.Command("helm", "repo", "update")); err != nil {
-			return err
-		}
-	}
-
-	if err := RunAndIgnoreOutput(exec.Command(
-		"helm",
-		"install",
-		"cert-manager",
-		"jetstack/cert-manager",
-		"--namespace",
-		"cert-manager",
-		"--create-namespace",
-		"--version",
-		certmanagerVersion,
-		"--set",
-		"installCRDs=true",
-		"--timeout",
-		"5m",
-	)); err != nil {
-		return err
-	}
-
-	// Wait for cert-manager-webhook to be ready, which can take time if cert-manager
-	// was re-installed after uninstalling on a cluster.
-	if err := RunAndIgnoreOutput(
-		exec.Command(
-			"kubectl",
-			"wait",
-			"deployment.apps/cert-manager-webhook",
-			"--for",
-			"condition=Available",
-			"--namespace",
-			"cert-manager",
-			"--timeout",
-			"5m",
-		)); err != nil {
-		return err
-	}
-	if err := RunAndIgnoreOutput(
-		exec.Command(
-			"kubectl",
-			"wait",
-			"deployment.apps/cert-manager-cainjector",
-			"--for",
-			"condition=Available",
-			"--namespace",
-			"cert-manager",
-			"--timeout",
-			"60s",
-		)); err != nil {
-		return err
-	}
-	if err := RunAndIgnoreOutput(
-		exec.Command(
-			"kubectl",
-			"wait",
-			"deployment.apps/cert-manager-cainjector",
-			"--for",
-			"condition=Available",
-			"--namespace",
-			"cert-manager",
-			"--timeout",
-			"60s",
-		)); err != nil {
-		return err
-	}
-	return nil
-}
-
-func UninstallCertManagerIfApplicable(certManagerHasBeenInstalled bool) {
-	if certManagerHasBeenInstalled {
-		By("uninstalling the cert-manager bundle")
-		uninstallCertManager()
-	} else {
-		fmt.Fprint(GinkgoWriter,
-			"Note: The e2e test suite did not install cert-manager, thus it will also not uninstall it.\n",
-		)
-	}
-}
-
-func uninstallCertManager() {
-	if err := RunAndIgnoreOutput(exec.Command(
-		"helm",
-		"uninstall",
-		"cert-manager",
-		"--namespace",
-		"cert-manager",
-		"--ignore-not-found",
-	)); err != nil {
-		warnError(err)
-	}
-
-	if err := RunAndIgnoreOutput(
-		exec.Command(
-			"kubectl", "delete", "namespace", "cert-manager", "--ignore-not-found")); err != nil {
-		warnError(err)
-	}
-}
-
```

Comment:
nit: line 33 (certmanagerVersion declaration) is unused now and can be removed

Reply:
8f55734